specs/brew.ts

```diff
@@ -1,19 +1,20 @@
-var generators = {
-
+const brewGenerators: Record<string, Fig.Generator> = {
     servicesgenerators: {
         script: "brew services list | sed -e 's/ .*//' | tail -n +2",
-        postProcess: function (out) {
-            return out.split('\n').filter((line) => {
-                return !line.includes('unbound') && {
-                    name: line,
-                    type: "option"
-                }
-            })
+        postProcess: function (output) {
```

Comment:
I would go with arrow functions since we are not anymore in vanilla JS:
```suggestion
        postProcess: output => {
```